### ISSUE TITLE
[Broken Website] Google search results while logged in

### ISSUE THREAD

**Issue by Steve-luo** [darkreader/darkreader#5824]
# Broken Website

## Website

https://www.google.com/search?q=dark+reader

## How to Reproduce

 - Search any string on google.com

## Expected behavior

All elements are converted to dark

## Actual behavior

Header, footer and some content are not converted correctly

## Screenshots

<img width="2304" alt="Screen Shot 2021-05-15 at 03 09 06" src="https://user-images.githubusercontent.com/8736917/118354198-825aa100-b52f-11eb-9c3c-8b7324cd4d49.png">
<img width="2304" alt="Screen Shot 2021-05-15 at 03 10 04" src="https://user-images.githubusercontent.com/8736917/118354200-838bce00-b52f-11eb-8f40-1ec2511f26d9.png">
[Screen Recording](https://streamable.com/lv9jbq)

## System Information

- OS: macOS 11.3.1 (20E241)
- Browser: Safari 14.1 (16611.1.21.161.6)
- Dark Reader Version: 1.4.3

**Comment by jgspratt** [comment]
I had a similar issue, but it turned out to be a Stylus style that had broken ("Google Clean Dark").  After disabling that style, it worked correctly again.

Do you happen to have Stylus or another site-specific style plugin?

**Comment by AndrewDaGuy** [comment]
> Do you happen to have Stylus or another site-specific style plugin?

Maybe the Cascadea safari plugin?
![image](https://user-images.githubusercontent.com/66189242/119082700-23dd6900-b9ee-11eb-92df-74d8001578de.png)

**Comment by Steve-luo** [comment]
@jgspratt @AndrewDaGuy Nope. I only have Dark Reader and AdGuard that can modify web pages.

**Comment by AndrewDaGuy** [comment]
> @jgspratt @AndrewDaGuy Nope. I only have Dark Reader and AdGuard that can modify web pages.

Since safari's DR plugin is closed source, it looks like a safari issue :/
